open-ce/test_feedstock.py

```diff
@@ -168,7 +186,8 @@ def gen_test_commands(test_file=utils.DEFAULT_TEST_CONFIG_FILE, variants=None, w
                                      conda_env=conda_env,
                                      create_env=True,
                                      working_dir=working_dir))
-
+    if not test_data['tests']:
```

Comment:
This could probably be combine with the check on line 176. No need to create conda environments if there aren't any tests.

Reply:
That's a good idea. I updated that and the test.